brave-spancollector-scribe/src/main/java/com/github/kristofa/brave/scribe/EmptyScribeSpanCollectorNotificationsHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.github.kristofa.brave.scribe;
+
+import com.twitter.zipkin.gen.Span;
+
+/**
+ * Empty implementation ignoring all notifications.
+ */
+class EmptyScribeSpanCollectorNotificationsHandler implements ScribeSpanCollectorNotificationsHandler {
+
+    @Override
+    public void onQueueSizeChanged(int currentQueueSize) {
```

Comment:
I'd prefer this to be more explicit. Ex instead of the somewhat vague of queue-size changed, make this an event:
onSpanAccepted, onSpanProcessed?